data/pokemon.json

```diff
@@ -362,7 +385,8 @@
     ]
   },
   {
-    "name": "graveler"
+    "name": "graveler",
+    "shiny": true
```

Comment:
Graveler can't be shiny if going by the usual pattern (not that it's ever been a raid boss nor do I expect it to be one)